polkadot-parachain/Cargo.toml

```diff
@@ -92,6 +92,11 @@ substrate-build-script-utils = { git = "https://github.com/paritytech/substrate"
 assert_cmd = "2.0"
 nix = "0.24"
 tempfile = "3.3.0"
+tokio = { version = "1.17.0", features = ["macros", "time", "parking_lot"] }
```

Comment:
@ruseinov 
other cumulus tokio deps, are `tokio = { version = "1.19.2" ...`, 
could we have it here the same?

Reply:
It's a dev-dependency for tests, I don't believe that it matters much, but sure!